src/Backend/Modules/ContentBlocks/Actions/Delete.php

```diff
@@ -31,19 +32,8 @@ public function execute(): void
         }
         $deleteFormData = $deleteForm->getData();
 
-        /** @var ContentBlockRepository $contentBlockRepository */
-        $contentBlockRepository = $this->get('content_blocks.repository.content_block');
-
-        $contentBlock = $contentBlockRepository->findOneByIdAndLocale(
-            $deleteFormData['id'],
-            Locale::workingLocale()
-        );
-
-        if ($contentBlock === null) {
-            $this->redirect(BackendModel::createURLForAction('Index', null, null, ['error' => 'non-existing']));
-
-            return;
-        }
+        /** @var ContentBlock $contentBlock */
+        $contentBlock = $this->getContentBlock((int) $deleteFormData['id']);
```

Comment:
Disagree, I think it's a good practice to send the right types. Looks a bit weird to send a string where an int is required.